rpm/droidmedia.spec

```diff
@@ -83,10 +101,12 @@ cp out/target/product/*/system/bin/minisfservice \
 cp external/droidmedia/*.h $RPM_BUILD_ROOT/%{_includedir}/droidmedia/
 cp external/droidmedia/hybris.c $RPM_BUILD_ROOT/%{_datadir}/droidmedia/
 
-%files
+LIBDMSOLOC=$RPM_BUILD_ROOT/file.list
+echo %{_libexecdir}/droid-hybris/system/$DROIDLIB/libdroidmedia.so > %{LIBDMSOLOC}
+echo %{_libexecdir}/droid-hybris/system/$DROIDLIB/libminisf.so > %{LIBDMSOLOC}
```

Comment:
I's this place file.list wil be overwritten & will contain only first record, change to 
echo %{_libexecdir}/droid-hybris/system/$DROIDLIB/libdroidmedia.so >> %{LIBDMSOLOC}
echo %{_libexecdir}/droid-hybris/system/$DROIDLIB/libminisf.so >> %{LIBDMSOLOC}

Reply:
thanks!

Reply:
https://github.com/sailfishos/droidmedia/pull/37/commits/e22e78f3ae925ad8521deb540e2d1c37ab979733#diff-fbbee40d360670af517afd4b68cc3cb1R105
it has already been changed